FrontEndLib/Sound.cpp

```diff
@@ -985,7 +985,9 @@ SOUNDSTREAM* CSound::LoadSongStream(const WSTRING& wstrSongFilepath, UINT mode)
 	WideCharToMultiByte(CP_ACP, 0, wstrSongFilepath.c_str(), -1, sANSI, 512, NULL, NULL);
 #else
 	char sANSI[wstrSongFilepath.length() + 1];
-	UnicodeToUTF8(wstrSongFilepath.c_str(), sANSI);
+	string sANSIString;
```

Comment:
Same here.  Let's just replace the sANSI char[] buffer with a string so we don't need to go through these code contortions.